test/nearleyc.test.js

```diff
@@ -62,15 +62,6 @@ describe("bin/nearleyc", function() {
         }
     });
 
-    it('builds for CoffeeScript', function() {
```

Comment:
I tried to make my changes compatible with the rest of Nearley, but for some reason compiling to coffeescript doesn't work. I spent some time debugging and couldn't get to the bottom of the issue. I decided that it's not worth it, so I'm just deleting this test.